### DISCUSSION TITLE
Can't capitalize collection or field names

### DISCUSSION THREAD

**Discussion by jrgarcia11** [pocketbase/pocketbase#30] [Q&A]
I'm attempting to enter field name while creating a collection, but the first letter of the field can't be capitalized. Is this to enforce camel case? I'm using linux version.

**Comment by jrgarcia11** [comment]
Also cannot capitalize first letter of the Collection name

**Comment by ganigeorgiev** [comment]
This is done intentionally only in the admin UI to ensure consistency across the user defined api.

Technically, it is allowed to capitalize collection name and fields.

I can remove the restriction from the UI, but Is there any particular reason you want to capitalize them or it is just a preference?

**Comment by jrgarcia11** [comment]
I suppose it's a preference. It was unclear why I couldn't capitalize the names because I expect it to match my input.

**Comment by ganigeorgiev** [comment]
I'd leave the issue open to see what the other users are thinking, but for me it is better to force some conventions, because the collection name and schema fields are used in the api response where we already have lowercase system fields like `id`, `created`, `updated`.

**Comment by rangzen** [comment]
It seems that fields can't have a mix either in the name. "fieldName" will not be retrieved with a structure with a public field like "FieldName" or "Fieldname". I have the problem right now. If I rename to "fieldname" then it works. @ganigeorgiev can you confirm? Is there a good practice for composed names like PilotName? pilot-name? pilot_name? or just concatened pilotname? Thank you

**Reply by ganigeorgiev** [reply]
I'm not sure that I follow. Please open a new discussion with more details about your use case and what you are trying to do.
